### PR TITLE
Add multiple inheritance group

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

File: test/autorest/migroup/migroup_test.go [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+import (
+	"context"
+	"generatortests/autorest/generated/migroup"
+	"generatortests/helpers"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+
+func getIntegerOperations(t *testing.T) migroup.MultipleInheritanceServiceClientOperations {
```
Comment (catalinaperalta) [inline test/autorest/migroup/migroup_test.go:15]: getMultipleInheritanceOperations

File: test/autorest/migroup/migroup_test.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	client, err := migroup.NewDefaultClient(nil)
+	if err != nil {
+		t.Fatalf("failed to create client: %v", err)
+	}
+	return client.MultipleInheritanceServiceClientOperations()
+}
+
+// GetCat - Get a cat with name 'Whiskers' where likesMilk, meows, and hisses is true
+func TestGetCat(t *testing.T) {
+	client := getIntegerOperations(t)
```
Comment (catalinaperalta) [inline test/autorest/migroup/migroup_test.go]: all of these will need to be updated once the function name is updated

File: test/autorest/migroup/migroup_test.go (hunk 1 above)
Comment (jhendrixMSFT) [reply test/autorest/migroup/migroup_test.go]: fixed

File: test/autorest/migroup/migroup_test.go (hunk 2 above)
Comment (jhendrixMSFT) [reply test/autorest/migroup/migroup_test.go]: fixed

**Pull request merged**